src/drivers/github.ts

```diff
@@ -0,0 +1,118 @@
+import { defineDriver } from './utils'
+import { $fetch } from 'ohmyfetch'
+import { withTrailingSlash } from 'ufo'
+
+export interface GithubOptions {
+  repo: string
+  /**
+   * @default "main"
+   */
+  branch: string
+  /**
+   * @default ""
+   */
+  dir: string
+  /**
+   * @default 600
+   */
+  ttl: number
+  token?: string
+}
+
+const GITHUB_URL = 'https://api.github.com'
+const GITHUB_CDN_URL = 'https://raw.githubusercontent.com'
+
+export default defineDriver((_opts: GithubOptions) => {
+  const opts = { branch: 'main', ttl: 600, ..._opts }
```

Comment:
I think we can keep default object outside. Also, default dir is missing?

Reply:
Applied, do you mind double check in this?